plugins/Marketplace/resources/qml/ManagePackagesButton.qml

```diff
@@ -18,7 +18,7 @@ Button
     background: Rectangle
     {
         color: backgroundColor
-        border.color: transparent
+        border.color: "transparent"
```

Comment:
So like no border? Why not just remove the border.color all together?

Reply:
I assumed that the one introducing this had some sort of reason originally. I'll remove it to see what happens tomorrow.

Reply:
I think I introduced it with some ctrl-c, ctrl-v action

Reply:
Ah, so it must've been an actual constant/variable originally.

Reply:
In any case, fixed in https://github.com/Ultimaker/Cura/pull/10812/commits/6df6dab6f01db427a9b73f1012a2d5f28fd9c3fb